common/src/main/java/com/wynntils/screens/discoveries/WynntilsDiscoveriesScreen.java

```diff
@@ -255,9 +267,26 @@ public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialT
     }
 
     private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
-        if (!(this.hovered instanceof TooltipProvider tooltipWidget)) return;
+        List<Component> tooltipLines = new ArrayList<>();
+
+        if (this.hovered instanceof SortOrderWidget) {
+            switch (contentSortOrder) {
+                case LEVEL -> tooltipLines = List.of(
```

Comment:
Several comments. 

The i18n name `screens.wynntils.wynntilsQuestBook` is not really correct here. If it is shared between screens, maybe it need a more generic name?

This is borderline close to begging to be generalized. I'm not sure it should be the role of ContentSortOrder to know how it should be displayed, but ... maybe ..? At least the i18n names seem to be derivable from the sort order.

Reply:
Fixed.